requirements.txt

```diff
@@ -3,3 +3,5 @@ langchain>=0.0.193
 duckpy
 huggingface_hub
 pytz
+fschat==0.2.13
+git+https://github.com/DachengLi1/LongChat
```

Comment:
should we stick to a fixed working commit for LongChat which is also compatible with fschat==0.2.13?